src/components/mdx/code-block.js

```diff
@@ -81,30 +96,42 @@ const CodeBlock = props => {
   return (
     <Highlight {...defaultProps} code={code} language={language}>
       {({ className, style, tokens, getLineProps, getTokenProps }) => (
-        <pre className={codeBlockClassName(theme) + " CodeBlock--language-" + language} language={language}>
+        <pre
+          className={
+            codeBlockClassName(theme) + " CodeBlock--language-" + language
+          }
+          language={language}
+        >
           {codeFrontmatter.header && (
             <span className="CodeBlock--header">{codeFrontmatter.header}</span>
           )}
           {codeFrontmatter.filename && !codeFrontmatter.header && (
-            <span className="CodeBlock--filename">{codeFrontmatter.filename}</span>
+            <span className="CodeBlock--filename">
+              {codeFrontmatter.filename}
+            </span>
           )}
           <code>
             <span className="CodeBlock--rows">
               <span className="CodeBlock--rows-content">
                 {tokens.map((line, i) => (
                   <span
                     key={i}
-                    className={"CodeBlock--row" + (
-                      codeFrontmatter.highlight &&
+                    className={
+                      "CodeBlock--row" +
+                      (codeFrontmatter.highlight &&
                       codeFrontmatter.highlight.length &&
-                      codeFrontmatter.highlight.includes(i + 1) ?
-                        " CodeBlock--row-is-highlighted" : ""
-                    )}
+                      codeFrontmatter.highlight.includes(i + 1)
+                        ? " CodeBlock--row-is-highlighted"
+                        : "")
+                    }
                   >
                     <span className="CodeBlock--row-indicator"></span>
                     <span className="CodeBlock--row-content">
                       {addNewlineToEmptyLine(line).map((token, key) => (
```

Comment:
Another solution seems to be to simply add a `<br>` tag to each line inside of a plan token span:

```jsx
<span className="CodeBlock--row-content">
  {line.map((token, key) => (
    <span key={key} {...tokenProps(getTokenProps({ token, key }))}/>
  ))}
  {/* Forced newline added to the markup here for each line */}
  {/* This doesn't add an additional <br> but makes copying */}
  {/* And pasting more consistent across browsers */}
  <span className="CodeBlock--token-plain"><br /></span>
</span>
```

This way, you can totally remove the add